source/Symbiont.test.cc

```diff
@@ -538,3 +538,37 @@ TEST_CASE("Symbiont ProcessResources"){
     }
 
 }
+
+TEST_CASE("Symbiont growOlder"){
+    emp::Ptr<emp::Random> random = new emp::Random(-1);
+    SymWorld w(*random);
+    w.Resize(2,2);
+    SymConfigBase config;
+    config.SYM_AGE_MAX(2);
+
+    WHEN ("A free-living symbiont reaches its maximum age"){
+      config.FREE_LIVING_SYMS(1);
+      Symbiont * s = new Symbiont(random, &w, &config, 1);
+      w.AddOrgAt(s, 1);
+      THEN("The symbiont dies and gets removed from the world"){
+        REQUIRE(w.GetNumOrgs() == 1);
+        REQUIRE(s->GetDead() == false);
+        w.Update(); //sym goes from age 1->2
```

Comment:
If we had a getAge method in the symbiont, that would be helpful to check here that the age goes up correctly